Formula/kwok.rb

```diff
@@ -0,0 +1,34 @@
+class Kwok < Formula
+  desc "Simulates thousands of Nodes and Clusters"
+  homepage "https://kwok.sigs.k8s.io"
+  version "0.1.0"
+  url "https://github.com/kubernetes-sigs/kwok/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "f8b5d01df09e1b68078c9754bb1922f1e9be3794fd519cd0de615280a0fff84d"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "bash" => :build
+  depends_on "make" => :build
+
+  def install
+    system "make", "build", "VERSION=v#{version}"
+
+    arch = "arm64"
+    platform = "darwin"
+  
+    on_intel do
+      arch = "amd64"
+    end
+  
+    on_linux do
+      platform = "linux"
+    end
+    bin.install "bin/#{platform}/#{arch}/kwok"
+    bin.install "bin/#{platform}/#{arch}/kwokctl"
```

Comment:
```suggestion
    bin.install "bin/#{OS.kernel_name.downcase}/#{Hardware::CPU.arch}/kwok"
    bin.install "bin/#{OS.kernel_name.downcase}/#{Hardware::CPU.arch}/kwokctl"
```

Reply:
> Error: An exception occurred within a child process:
    Errno::ENOENT: No such file or directory - bin/linux/x86_64/kwok

`Hardware::CPU.arch` looks like the value is not amd64 but x86_64, is there another way to get amd64?